src/tools/rustbook/src/main.rs

```diff
@@ -46,23 +50,35 @@ fn main() {
 
                 ::std::process::exit(101);
             }
-        },
+        }
         ("linkcheck", Some(sub_matches)) => {
             if let Err(err) = linkcheck(sub_matches) {
                 eprintln!("Error: {}", err);
 
+                // HACK: ignore timeouts
+                #[allow(unused_mut)]
+                let mut actually_broken = false;
+
                 #[cfg(feature = "linkcheck")]
                 {
                     if let Ok(broken_links) = err.downcast::<BrokenLinks>() {
                         for cause in broken_links.links().iter() {
                             eprintln!("\tCaused By: {}", cause);
+
+                            if cause.contains("timed out") {
```

Comment:
Perhaps missing negation here?
```suggestion
                            if !cause.contains("timed out") {
```